src/components/DockerInstall.vue

```diff
@@ -64,7 +64,7 @@
       >
         <q-space />
         <div class="quickstart-container">
-          <a target="_blank" rel="noopener" id="macos-quickstart-link" href="https://docs.yugabyte.com/preview/quick-start/explore/ysql/#docker">Complete Docs</a>
+          <a target="_blank" rel="noopener" id="macos-quickstart-link" href="https://docs.yugabyte.com/preview/develop/build-apps/">Complete Docs</a>
```

Comment:
Build apps? Not quite the same... there's this https://docs.yugabyte.com/preview/sample-data/retail-analytics/

Reply:
changed